charmhelpers/contrib/openstack/context.py

```diff
@@ -781,6 +781,18 @@ def __call__(self):
         ensure_packages(['ceph-common'])
         return ctxt
 
+    def context_complete(self, ctxt):
+        """Override to ...
```

Comment:
I was hoping you'd expand the "..." :)  -- it was nice text you'd written in the last commit.

Reply:
How's that ;-)